app/src/components/market/common/xdai_bridge_modal/bridge_transaction_state.tsx

```diff
@@ -31,7 +30,7 @@ const SvgWrapper = styled.div`
   margin-bottom: 25px;
 `
 const BoldedText = styled.div`
-  color: ${({ theme }) => theme.colors.textColorDark};
+  color: ${props => props.theme.text1};
```

Comment:
here the same

Reply:
Here there is a big misunderstanding, my fault, but that seemed so logical/coherent to me. I believed I had to not only edit the theme import but also edit the colors when needed with the ones @pimato had set finals... I will revert all the colors modification and keep that for a later issue